nightingaleproject/canary
# Provide message resp options

Added response messages to the results page for "Producing FHIR VRDR Messages" and "Connectathon FHIR VRDR Messages(Producing)"

To test:
- Run a test in "Producing FHIR VRDR Messages"
- On the results page, click through the message response types and make sure each message appears in the box
- Repeat for "Connectathon FHIR VRDR Messages (Producing)"
- Repeat but select "Void" as the message type, make sure there's only an "ACK" or "Error" message option on the results page
